user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Tutorials/Tutorial_Apps_Managing_Pages.md

```diff
@@ -1,98 +1,130 @@
+   <H2 style="color:#0d5951; font-family:'Segoe UI Light','Segoe UI Web Light','Segoe UI Web Regular','Segoe UI','Segoe UI Symbol',HelveticaNeue-Light,'Helvetica Neue',Arial,sans-serif;margin:0;font-weight:100">
+   Manage all your IP addresses
+   </H2>
+   ```
 
-> [!NOTE]
-> While feed references are maintained when you duplicate a page, other pages that consume feeds from the duplicated page will only receive values from the original page.
+1. In the *Layout* tab, set the theme of the component to "Transparent".
 
-1. To **duplicate** a page, in the navigation sidebar, click the ... icon next to the page name and select *Duplicate*. Create a duplicate of the "IP addresses" page.
+   ![Transparent theme](~/user-guide/images/ComponentTransparentTheme.png)
 
-  ![Duplicate option](~/user-guide/images/PageDuplicate.png)
+1. In the *Data* pane, configure a GQI query that uses the ad hoc data source *IP addresses - Dummy*.
 
-  This will create a copy of the page. You can then customize the duplicated page to suit your requirements.
+   This is a data source that was added to your system when you deployed the package in step 1.
+
+   ![IP address query](~/user-guide/images/IPAddressesQuery.png)
+
+1. Add a table component to the page and configure it to show the query you have just configured.
+
+## Step 5: Duplicate a page
+
+You can quickly create a duplicate of an existing page, resulting in a new page that is identical to the original. The entire configuration, including feed references, will be preserved in this duplicated page.
+
+1. In the navigation sidebar, click the ... icon next to the page name of your new page and select *Duplicate*.
+
+   ![Duplicate option](~/user-guide/images/PageDuplicate.png)
+
+   This will create a copy of the page. You can then customize the duplicated page to suit your requirements.
 
 1. Change the page name to "Subnets" and use the "Switch" icon.
-2. Change the content of the web component. Remove the query from the table and create a new query that fetches data from the "IP subnets - Dummy" ad hoc data source instead.
 
-### (Opional) Reordering, hiding and deleting pages
+1. Remove the query from the table and create a new query that fetches data from the *IP subnets - Dummy* ad hoc data source instead.
+
+## Step 6: (Optional) Reorder, hide, and delete pages
+
+1. Try changing the order of the pages in your app: Hover over a page until you see up and down arrows and then click those arrows as necessary.
 
-- To **change the order** of the pages in your app, including the initial page (i.e. the one located at the top of the navigation sidebar), hover over a page until you see up and down arrows and then click those arrows as necessary.
+   ![Reordering pages](~/user-guide/images/PageReorder.png)
 
-  ![Reordering pages](~/user-guide/images/PageReorder.png)
+1. Add another new page to the app.
```

Comment:
Not really required, but can't hurt as well.

Reply:
I added this so they wouldn't have to delete the nice pages they added earlier.